dotCMS/src/integration-test/java/com/dotcms/graphql/business/GraphqlAPITest.java

```diff
@@ -782,12 +786,10 @@ public void testGetSchema_GivenFailuresInRelationshipField_SchemaShouldStillGene
 
             APILocator.getGraphqlAPI().invalidateSchema();
 
-            RelationshipAPI relationshipAPI = Mockito.mock(RelationshipAPI.class);
-            Mockito.when(relationshipAPI.
-                    getRelationshipFromField(relationshipField, APILocator.systemUser()))
-                    .thenReturn(null);
+            // this mock relationship api will produce errors when generating the rel field
+            setMockRelationshipAPI(relationshipField);
 
-            final GraphQLSchema schema = new GraphqlAPIImpl(relationshipAPI).getSchema();
+            GraphQLSchema schema = APILocator.getGraphqlAPI().getSchema();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'schema' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642169)